settings/UL/settings_embeddingselection_2016preVFPUL.yaml

```diff
@@ -79,11 +106,27 @@ ID_pt_eta_bins:
   ratio_y_range: [0.8, 1.2]
 
 EmbID_pt_bins_inc_eta:
-  # Settings for histogram creation
   name: "EmbID_pt_bins_inc_eta"
   var: "m_vis(100,50,150)"
-  tag: "(trg_IsoMu22_tag || trg_IsoTkMu22_tag || trg_IsoMu22_eta2p1_tag || trg_IsoTkMu22_eta2p1_tag) && fsr_photon_veto_tag < 0.5 && fsr_photon_veto_probe < 0.5 && pt_tag > 23 && iso_tag < 0.15 && dz_tag < 0.2 && dxy_tag < 0.045 && id_medium_tag"
-  probe: "is_global_probe && id_loose_probe"
+  tag:  (
+          (iso_tag < 0.15) && 
+          ((fsr_photon_veto_tag < 0.5) && (fsr_photon_veto_probe < 0.5)) && 
+          (id_medium_tag) &&
+          (
+            (trg_IsoMu22_tag) || 
+            (trg_IsoTkMu22_tag) || 
+            (trg_IsoMu22_eta2p1_tag) || 
+            (trg_IsoTkMu22_eta2p1_tag)
+          ) && 
+          (pt_tag > 23.0) &&
+          ((q_1 + q_2) == 0) &&
+          (abs(dz_tag) < 0.2) &&
+          (abs(dxy_tag) < 0.045)
+        )
+  probe:  (
+            (is_global_probe) && 
+            (id_medium_probe)
```

Comment:
wait, why is is medium here ? In the embedding selection we use 

`is_loose` and `is_global`: https://github.com/KIT-CMS/cmssw/blob/embedding_UL_10_6_28/TauAnalysis/MCEmbeddingTools/python/SelectingProcedure_cff.py#L49-L53

Reply:
Just a bit of clarification: I used medium ID back in the time to be consistent with Imperial College measurement because  and according to this DP note https://cds.cern.ch/record/2257968/files/DP2017_007.pdf muon ID performance doesn't get significantly worse while changing more tighter working point. But since `is_loose` working point was used at the selection step during production it's better to change it back to `is_loose` from `is_medium`

Reply:
But there is nothing embedding-related in the measurements that you link. The embedding selection sfs have to fit the selection used during embedding production so using `is_medium` is just not correct